_maps/map_files/generic/CentCom.dmm

```diff
@@ -2307,9 +2307,9 @@
 /area/centcom/central_command_areas/briefing)
 "gG" = (
 /obj/effect/turf_decal/stripes/box,
-/obj/machinery/portable_atmospherics/pump{
+/obj/machinery/portable_atmospherics/pump/lil_pump{
 	desc = "A betrayer to pump-kind.";
```

Comment:
```suggestion
/obj/machinery/portable_atmospherics/pump/lil_pump{
	desc = "A betrayer to pump-kind."
}
```
The map key appears to have screwed up during your edit, just remove the semicolon and ensure that there are no newlines within the variables associated to an object.

Reply:
Copilot has fucked me.